Makefile

```
@@ -10,8 +10,37 @@ PLATFORM ?= linux/amd64,linux/arm64
+	@echo "Rust Development:"
+	@grep -E '^(test|fmt|lint|clean-rust).*:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}'
+	@echo ""
+	@echo "Kubernetes:"
+	@grep -E '^(cluster-|deploy-).*:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}'
+	@echo ""
+	@echo "Other targets:"
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | grep -v -E '^(tilt-|dev-|build|push|test|fmt|lint|clean-rust|cluster-|deploy-)' | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}'
+
+# Development targets
+.PHONY: dev-up
+local: ## Start development environment
```

checkmake (0.2.2)

[warning] Target body for "local" exceeds allowed length of 5 (9).

(maxbodylength)